---
title: Welcome to MBL STAMPS!
feature_image: "/images/stamps2018.jpg"
excerpt: "Home page for the MBL STAMPS course."
---

<a href="https://www.mbl.edu/"><img src="/images/MBL-logo.jpg" style="float: right; height: 150px; padding-right: 90px; padding-top: 25px; margin: 20px"></a>

#### Course Information  
**2022 dates:** 21 July, 2022 – 30 July, 2022  


**Course Directors**  
[Mihai Pop](http://www.cbcb.umd.edu/~mpop/) ([University of Maryland](https://www.umd.edu/))  
Tracy Teal ([RStudio](https://rstudio.com/))  

**Financial and Application Information**  
Please see the [MBL STAMPS course page](https://www.mbl.edu/education/courses/stamps/) for course logistics information. Applications are closed for 2022.

**2022 Course Schedule and Materials**
Our course schedule and materials for the course are available on the [2022 STAMPS Course page](https://github.com/mblstamps/stamps2022/wiki).


#### What is STAMPS?
The STAMPS course (**S**trategies and **T**echniques for **A**nalyzing **M**icrobial **C**ommunity **P**opulation **S**tructures) takes places at the [Marine Biological Laboratory](https://www.mbl.edu/) in Woods Hole, MA, USA each summer.  

Current sequencing technologies enable highly comprehensive investigations of microbial communities. But the size of these datasets poses enormous computational challenges, and our ability to generate them has largely been outpacing our collective ability to manipulate and utilize them. The STAMPS course promotes dialogue and the exchange of ideas between experts in environmental and microbiome analysis and offers interdisciplinary bioinformatics and statistical training to practitioners of molecular microbial ecology and genomics.  

The course is designed for established investigators, postdoctoral fellows, and advanced graduate students from diverse biological fields. Topics to be covered include but are not limited to: acquisition and organization of next-generation sequence data; principles of quality control and data management; processing and analyzing marker-gene/amplicon data (such as 16S sequencing); assembly and annotation of shotgun metagenomic data; statistical models for estimating microbial diversity; and microbial community comparison methodology. The course additionally covers the basics for working in the Unix command-line and R statistical environments, but some prior experience with these is strongly recommended. Guided by developers of tools such as BreakAway, DivNet, GToTree, PhyloSeq, SourMash, and QIIME2, participants of the STAMPS course will have the opportunity to try various analysis techniques and discuss their own data and analyses with faculty.

---

#### 2022 Course Faculty
[Mihai Pop](http://mpop.umiacs.io/), University of Maryland  
Tracy Teal, RStudio  
[Titus Brown](http://ivory.idyll.org/lab/), University of California, Davis  
[Ben Callahan](https://callahanlab.cvm.ncsu.edu/), North Carolina State University  
[Curtis Huttenhower](https://huttenhower.sph.harvard.edu/), Harvard University
[Mike Lee](https://astrobiomike.github.io/research/), NASA Ames Research Center  
[Todd Treangen](https://csweb.rice.edu/todd-treangen), University of Maryland  
[Amy Willis](http://statisticaldiversitylab.com/), University of Washington  



