---
title: Welcome to MBL STAMPS 2019!
feature_image: "/images/stamps2018.jpg"
excerpt: "Home page for the MBL STAMPS 2019 course."
---

<a href="https://www.mbl.edu/"><img src="/images/MBL-logo.jpg" style="float: right; height: 150px; padding-right: 90px; padding-top: 25px; margin: 20px"></a>

#### Course Information
**Course Dates:** 22-July - 1-August-2019

**Deadline for applications:** 19-April-2019  
{% include button.html text="Apply Here" link="https://ws2.mbl.edu/studentapp/studentapp.asp?courseid=stamps" color="#006b6f" %}  

**Course Directors:**  
[Mihai Pop](http://www.cbcb.umd.edu/~mpop/) ([University of Maryland](https://www.umd.edu/))  
Tracy Teal ([The Carpentries](https://carpentries.org/))  

**Financial Information:**  
Please see [the application page](https://ws2.mbl.edu/studentapp/studentapp.asp?courseid=stamps) for financial information.  

#### What is STAMPS?
The STAMPS course (**S**trategies and **T**echniques for **A**nalyzing **M**icrobial **C**ommunity **P**opulation **S**tructures) takes places at the [Marine Biological Laboratory](https://www.mbl.edu/) in Woods Hole, MA, USA each summer.  

Current sequencing technologies enable highly comprehensive investigations of microbial communities. But the size of these datasets poses enormous computational challenges, and our ability to generate them has largely been outpacing our collective ability to manipulate and utilize them. The STAMPS course promotes dialogue and the exchange of ideas between experts in environmental and microbiome analysis and offers interdisciplinary bioinformatics and statistical training to practitioners of molecular microbial ecology and genomics.  

The course is designed for established investigators, postdoctoral fellows, and advanced graduate students from diverse biological fields. Topics to be covered include but are not limited to: acquisition and organization of next-generation sequence data; principles of quality control and data management; processing and analyzing marker-gene/amplicon data (such as 16S sequencing); assembly and annotation of shotgun metagenomic data; statistical models for estimating microbial diversity; and microbial community comparison methodology. The course additionally covers the basics for working in the Unix command-line and R statistical environments, but some prior experience with these is strongly recommended. Guided by developers of tools such as BreakAway, DivNet, GToTree, PhyloSeq, SourMash, and QIIME2, participants of the STAMPS course will have the opportunity to try various analysis techniques and discuss their own data and analyses with faculty.

---

#### 2019 Course Faculty
[Mihai Pop](http://mpop.umiacs.io/), University of Maryland  
Tracy Teal, The Carpentries  
[Yana Bromberg](https://bromberglab.org/), Rutgers University  
[Titus Brown](http://ivory.idyll.org/lab/), University of California, Davis  
[Ben Callahan](https://callahanlab.cvm.ncsu.edu/), North Carolina State University  
[Susan Holmes](http://statweb.stanford.edu/~susan/susan_person.html), Stanford University  
[Adina Howe](www.germslab.org), Iowa State University  
[Rob Knight](https://knightlab.ucsd.edu/), University of California, San Diego  
[Mike Lee](https://astrobiomike.github.io/research/), NASA Ames Research Center  
[Christian Mueller](https://www.simonsfoundation.org/team/christian-muller/), Simons Foundation  
[Todd Treangen](https://csweb.rice.edu/todd-treangen), University of Maryland  
[Tandy Warnow](http://tandy.cs.illinois.edu/), University of Illinois at Urbana-Champaign  
[Amy Willis](http://statisticaldiversitylab.com/), University of Washington  

Feel free to check out the schedule and lectures from last year located **[here](https://github.com/mblstamps/stamps2018/wiki#schedule)**!  

